src/app/content/components/Toolbar/index.tsx

```diff
@@ -94,13 +103,30 @@ class Toolbar extends React.Component<Props, State> {
     const hideFromFocus = this.props.tocOpen === true
       || (this.props.tocOpen === null && !this.props.searchSidebarOpen);
 
+    let searchButtonColor = null;
+
+    if (book) {
+      switch (searchButtonStyle) {
+        case 'grayButton':
```

Comment:
This could be written with a ternary, but I guess an even better option would be to have this written like:
```
const searchButtonColor = book ? getSearchButtonColor(searchButtonStyle, bookTheme) : null;
```
or maybe even better: create a selector called something like `searchButtonColorExperiment` that checks for `book`, `bookTheme` and `searchButtonStyle`